tests/testthat/test-rowClust.Rmd

```diff
@@ -0,0 +1,34 @@
+# Loading example data
+
+```{r, message=FALSE, warning=FALSE}
+library(mia)
+data("GlobalPatterns")
+tse <- GlobalPatterns
+tse <- agglomerateByRank(tse, rank="Class")
+source("../../R/rowClust.R")
```

Comment:
This source line is just for testing? Should be removed from ready made version,

Reply:
yes just for testing